add_catch(test_power_of_two test.cpp)
